pyensembl/compute_cache.py

```diff
@@ -8,7 +8,7 @@
 2) Cache warm on disk. Parse or unpickle the serialized result into memory.
 3) Cache warm in memory. Return cached object.
 """
-import cPickle
+import pickle
```

Comment:
I think the correct thing to here is to try to `import cPickle as pickle`, and if that fails with an `ImportError` then `import pickle` (and, since this will look insane, comment that we're doing it because GvR made us).